virt/xen/xen.py

```diff
@@ -0,0 +1,294 @@
+#!/usr/bin/env python
+
+import sys
+from time import time
+import XenAPI
+from XenAPI import Failure, NewMaster
+from collections import defaultdict, OrderedDict
+import virt
+
+class Guest(object):
```

Comment:
Is it even necessary to have it here? What is the difference between it and Guest class from virt/virt.py module?